### PR TITLE
[SubtitleFormat] - Update (SubViewer20.cs)

### PR DESCRIPTION
**Pull request by ivandrofly**
Fix issue present in previous [PR ](https://github.com/SubtitleEdit/subtitleedit/pull/1600)

- [x] - Ready for merge

### CODE REVIEW

File: libse/SubtitleFormats/SubViewer20.cs [hunk 1]
```diff
@@ -105,57 +95,73 @@ public override void LoadSubtitle(Subtitle subtitle, List<string> lines, string
...
-                    if (expecting == ExpectingLine.Text)
+                    catch
                     {
-                        if (line.Length > 0)
-                        {
-                            string text = line.Replace("[br]", Environment.NewLine);
-                            text = text.Replace("{\\i1}", "<i>");
-                            text = text.Replace("{\\i0}", "</i>");
-                            text = text.Replace("{\\i}", "</i>");
-                            text = text.Replace("{\\b1}", "<b>'");
```
Comment (ivandrofly) [inline libse/SubtitleFormats/SubViewer20.cs]: Replace `<b>' => <b>`

File: libse/SubtitleFormats/SubViewer20.cs [hunk 2]
```diff
@@ -105,57 +95,73 @@ public override void LoadSubtitle(Subtitle subtitle, List<string> lines, string
...
+                    subtitle.Paragraphs.Add(paragraph);
+                    paragraph = new Paragraph();
+                    expecting = ExpectingLine.TimeCodes;
+                }
+                else if (lineTrimmed.Length > 0) // Do not count empty lines
+                {
+                    _errorCount++;
+                }
             }
+            // Do not include metadata
```
Comment (niksedk) [inline libse/SubtitleFormats/SubViewer20.cs]: Hm, error handling should be improved... try to see what happens on non subviewer 2 file

**Comment by ivandrofly** [comment]
I think we are done now... can be merged :) /cc @niksedk

**Comment by ivandrofly** [comment]
More patches will come (bugfix/updates)

**Pull request closed without merging**